docs/source/user_guide/settings.rst

```diff
@@ -127,45 +143,51 @@ Simulation Box
+      <td style="padding: 8px; border: 1px solid #ddd; overflow-wrap: break-word; white-space: normal;">Visible only if Morphology size defined by number of molecules. If set, an approximately cubic morphology is generated. If disabled, X and Y are required.</td>
+      <td style="padding: 8px; border: 1px solid #ddd; overflow-wrap: break-word; white-space: normal;">False</td>
+    </tr>
+    <tr>
+      <td style="padding: 8px; border: 1px solid #ddd; overflow-wrap: break-word; white-space: normal;">X / A</td>
+      <td style="padding: 8px; border: 1px solid #ddd; overflow-wrap: break-word; white-space: normal;">Half the box size in x direction in A. Box extends from -Lx to Lx.</td>
+      <td style="padding: 8px; border: 1px solid #ddd; overflow-wrap: break-word; white-space: normal;">100.0</td>
+    </tr>
+    <tr>
+      <td style="padding: 8px; border: 1px solid #ddd; overflow-wrap: break-word; white-space: normal;">Y / A</td>
+      <td style="padding: 8px; border: 1px solid #ddd; overflow-wrap: break-word; white-space: normal;">Half the box size in y direction in A. Box extends from -Ly to Ly. Recommended: Lx=Ly.</td>
+      <td style="padding: 8px; border: 1px solid #ddd; overflow-wrap: break-word; white-space: normal;">100.0</td>
+    </tr>
+    <tr>
+      <td style="padding: 8px; border: 1px solid #ddd; overflow-wrap: break-word; white-space: normal;">Z / A</td>
+      <td style="padding: 8px; border: 1px solid #ddd; overflow-wrap: break-word; white-space: normal;">Box size in z direction (deposition axis) in A. Visible only for Morphology size defined by box size.</td>
+      <td style="padding: 8px; border: 1px solid #ddd; overflow-wrap: break-word; white-space: normal;">200.0</td>
+    </tr>
+    <tr>
+      <td style="padding: 8px; border: 1px solid #ddd; overflow-wrap: break-word; white-space: normal;">Set total height for multilayer</td>
+      <td style="padding: 8px; border: 1px solid #ddd; overflow-wrap: break-word; white-space: normal;">To deposit multilayer systems (multiple Deposit steps) requires setting the same global height in all Deposit steps. Enable and set the value as described below.</td>
+      <td style="padding: 8px; border: 1px solid #ddd; overflow-wrap: break-word; white-space: normal;">False (unchecked)</td>
+    </tr>
+    <tr>
+      <td style="padding: 8px; border: 1px solid #ddd; overflow-wrap: break-word; white-space: normal;">Total Z / A</td>
+      <td style="padding: 8px; border: 1px solid #ddd; overflow-wrap: break-word; white-space: normal;">Visible only if "Set total height..." above is enabled. Defines the estimated total box height of your multilayer deposition. Needs to be the same value for all sequential Deposit modules.</td>
+      <td style="padding: 8px; border: 1px solid #ddd; overflow-wrap: break-word; white-space: normal;">200</td>
+    </tr>
+  </tbody>
     </table>
 
 Simulation Parameters
```

Comment:
One need to add one additional note or change the existing one:
If the first is opt:
Note: Simulation Parameters <whatever it is called>  will only be visible if "Show simulation parameters" (advanced) is checked in.